stacks/serverless/upload.prx.org.yml

```diff
@@ -110,34 +112,34 @@ Resources:
           MetricsEnabled: true
           ResourcePath: "/*"
   # Custom Domain
-  # UploadRestApiDomainName:
-  #   Type: "AWS::ApiGateway::DomainName"
-  #   Properties:
-  #     CertificateArn: !Ref WildcardCertificateArn
-  #     DomainName: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
-  # UploadRestApiBasePathMapping:
-  #   Type: "AWS::ApiGateway::BasePathMapping"
-  #   Properties:
-  #     # BasePath: 'prod'
-  #     DomainName: !Ref UploadRestApiDomainName
-  #     RestApiId: !Ref UploadRestApi
-  #     Stage: !If [CreateProductionResources, prod, prod] # TODO these shouldn't be the same
-  # UploadRestApiDomainNameRecordSetGroup:
-  #   Type: "AWS::Route53::RecordSetGroup"
-  #   Properties:
-  #     Comment: Record sets for upload signing API
-  #     HostedZoneName: prx.tech.
-  #     RecordSets:
-  #       - Type: AAAA
-  #         Name: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
-  #         AliasTarget:
-  #           DNSName: !GetAtt UploadRestApiDomainName.DistributionDomainName
-  #           HostedZoneId: Z2FDTNDATAQYW2
-  #       - Type: A
-  #         Name: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
-  #         AliasTarget:
-  #           DNSName: !GetAtt UploadRestApiDomainName.DistributionDomainName
-  #           HostedZoneId: Z2FDTNDATAQYW2
+  UploadRestApiDomainName:
+    Type: "AWS::ApiGateway::DomainName"
+    Properties:
+      CertificateArn: !Ref WildcardCertificateArn
+      DomainName: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
+  UploadRestApiBasePathMapping:
+    Type: "AWS::ApiGateway::BasePathMapping"
+    Properties:
+      # BasePath: 'prod'
+      DomainName: !Ref UploadRestApiDomainName
+      RestApiId: !Ref UploadRestApi
+      Stage: !If [CreateProductionResources, prod, stag]
+  UploadRestApiDomainNameRecordSetGroup:
+    Type: "AWS::Route53::RecordSetGroup"
+    Properties:
+      Comment: Record sets for upload signing API
+      HostedZoneName: prx.tech.
+      RecordSets:
+        - Type: AAAA
+          Name: !Join ["", ["upload-", !Ref UploadRestApi, "-", !Ref "AWS::Region", !If [CreateStagingResources, ".staging", ""], ".prx.tech"]]
+          AliasTarget:
+            DNSName: !GetAtt UploadRestApiDomainName.DistributionDomainName
+            HostedZoneId: Z2FDTNDATAQYW2
```

Comment:
We use `Z2FDTNDATAQYW2` in about 9 places in the templates, I wonder if it should be factored into a var of some kind?